staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -171,7 +171,7 @@ func LogRequestPatch(ctx context.Context, patch []byte) {
 // will be converted to the given gv.
 func LogResponseObject(ctx context.Context, obj runtime.Object, gv schema.GroupVersion, s runtime.NegotiatedSerializer) {
 	ac := AuditContextFrom(WithAuditContext(ctx))
-	if ac.GetEventLevel().Less(auditinternal.LevelMetadata) {
+	if ac.GetEventLevel().Less(auditinternal.LevelRequestResponse) {
```

Comment:
Before https://github.com/kubernetes/kubernetes/pull/129472, the LogResponseObject logged ResponseStatus for LevelMetada level and whole object when on LevelRequestResponse level. 

With this change we will no longer log ResponseStatus, see https://github.com/kubernetes/kubernetes/pull/129472/files#r2084736668

Reply:
@serathius i believe this moved here - https://github.com/kubernetes/kubernetes/pull/129472/files#diff-dd3112aa714e032debca6f992d693042b2b5cc54e49b30f409e1eedf00dcc856R155-R172

Reply:
Added test cases!

Reply:
please check if i should add any more tests @serathius

Reply:
> @serathius i believe this moved here - https://github.com/kubernetes/kubernetes/pull/129472/files#diff-dd3112aa714e032debca6f992d693042b2b5cc54e49b30f409e1eedf00dcc856R155-R172

Unless I'm missing something, that function is only called from this one, and now only if the log level is at least RequestResponse.

```suggestion
	if ac.GetEventLevel().Less(auditinternal.LevelMetadata) {
		return
	} else if ac.GetEventLevel().Less(auditinternal.LevelRequestResponse) {
		ac.LogResponseObject(status, nil)
```

Reply:
oh, this needs to move to after the status conversion, but you get the idea.

Reply:
ACK @tallclair

Reply:
Makes sense now!

Reply:
cool!

Reply:
thanks @tallclair